libs/watt/src/lib/components/expandable-card/watt-expandable-card.component.html

```diff
@@ -14,7 +14,7 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<mat-expansion-panel [expanded]="expanded">
+<mat-expansion-panel [expanded]="expanded" class="watt-expandable-card watt-elevation">
```

Comment:
issue: the same CSS classes are also added in the component's class, so they'll be doubled. Consider using [NgClass](https://angular.io/api/common/NgClass) to toggle between `watt-elevation`/`watt-solid` on the `mat-expansion-panel` element, while `watt-expandable-card` is always present.